en/pragma.tex

```diff
@@ -304,6 +304,16 @@ \subsubsection{} In casual conversation the reflexive perfective of
 \NTeri{3/8/2011}{http://naviteri.org/2011/08/new-vocabulary-clothing/}
 
 
+\subsubsection{Slang} In casual conversation some constructions that
+deviate from standard grammar may be used to serve certain purposes.
+For example, the mood infixes \N{<ei>} and \N{<äng}} may be inserted
```

Comment:
Is this meant to say [...] `\N{<äng>} may be inserted` ?

Reply:
Now I catch it... Srane, that's a mini typo :) Irayo!